.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @oleg-nenashev
```

Comment:
I just created @gradleup/admins. Can we use that instead?

Reply:
```suggestion
* @GradleUp/admins
```